test/mime_test.exs

```diff
@@ -4,6 +4,10 @@ defmodule MIMETest do
   import MIME
   doctest MIME
 
+  setup_all %{} do
+    Code.compiler_options(ignore_module_conflict: true)
```

Comment:
I think you can move this to the test_helper.exs?

Reply:
Done! ✅